### PR TITLE
new features. dns labels validation

### PR DESCRIPTION
**Pull request by vanzhiganov**

### CODE REVIEW

File: validators/dns.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+pattern = re.compile(
+    r'^((?![0-9]{1,63}[.])(?![0-9]+$)'
+    r'(?!-)([a-zA-Z0-9-_]{0,63}(?<!-)([.])?)){1,32}$'
+)
+
+
+@validator
+def label(value):
```
Comment (kvesteri) [inline validators/dns.py]: Would hostname made more sense as a name for this? Label sounds a bit too generic for me

File: tests/test_dns.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    'example.com',
+    'xn----gtbspbbmkef.xn--p1ai',
+    'underscore_subdomain.example.com',
+    'something.versicherung',
+    'example.com.',
+    '_srv._xmpp-server._tcp.example.com.',
+    '_srv._xmpp-server._tcp',
+
+])
+def test_returns_true_on_valid_domain(value):
```
Comment (kvesteri) [inline tests/test_dns.py]: Combine the tests and use same concept naming everywhere (I would prefer hostname).

**Comment by kvesteri** [comment]
Could you also squash the commits?

**Comment by vanzhiganov** [comment]
squash commit is not available for me

**Pull request closed without merging**